### PR TITLE
Adds the Capability of Enumerate Local Groups

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: discovery/d7add940-1137-4975-b7d5-cab9a107a6c6.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+id: d7add940-1137-4975-b7d5-cab9a107a6c6
+name: Enumerate Local Groups
+description: Retrieves all Local Groups
```
Comment (khyberspache) [inline discovery/d7add940-1137-4975-b7d5-cab9a107a6c6.yml:3]: Retrieve local group information on the target system.

File: discovery/d7add940-1137-4975-b7d5-cab9a107a6c6.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+metadata:
+  version: 1
+  author:
+    - w0rk3r
+  tags:
+    - Powerview
```
Comment (khyberspache) [inline discovery/d7add940-1137-4975-b7d5-cab9a107a6c6.yml:9]: We are going to keep tags empty for now on this repo unless myself/privateducky adds it. Make this 
`tags: []`

**Review by khyberspache — CHANGES_REQUESTED** [review]
very small changes

**Comment by w0rk3r** [comment]
Fixed

**Review by khyberspache — APPROVED** [review]

**Pull request merged**